packages/client/components/GitLabScopingSearchResults.tsx

```diff
@@ -94,16 +82,35 @@ const GitLabScopingSearchResults = (props: Props) => {
                     path
                   }
                   query {
-                    projects(membership: true, first: $count, after: $cursor)
-                      @connection(key: "GitLabScopingSearchResults_projects") {
+                    projects(
+                      membership: true
+                      first: $projectsFirst
+                      after: $projectsAfter
+                      sort: "latest_activity_desc"
+                      ids: null # $projectIds
```

Comment:
Originally, we discussed using the skip & include directives because the projects query would be null if `ids` is an empty array. However, when `ids` is `null`, the query works and we can successfully search issues. 

So, I suggest we don't use skip or include and just have one query that accepts the `ids` argument. If the user hasn't selected any projects, `ids` is null. If they have, then we have an array of project `ids`.

One problem that I've run into is when scrolling down the list, triggering the Intersection Observer, and grabbing the next results, the data returned from the query is `null`. I suspect that the `useLoadNextOnScrollBottom` code may need to be tweaked, but this can be done when implementing the project filter menu here: https://github.com/ParabolInc/parabol/issues/6046. For now, I've used `null` as a placeholder.